core/src/main/java/hudson/model/SlaveComputerConfiguration.java

```diff
@@ -0,0 +1,32 @@
+package hudson.model;
+
+import hudson.Extension;
+import jenkins.model.GlobalConfiguration;
+import net.sf.json.JSONObject;
+import org.jenkinsci.Symbol;
+import org.kohsuke.stapler.StaplerRequest;
+
+@Extension @Symbol("slaveComputer")
+public class SlaveComputerConfiguration extends GlobalConfiguration {
```

Comment:
I need to check whether there are other suitable configuration classes. Maybe it makes sense to just name it "RemotingConfiguration", mark as restricted and then detach to a plugin along with other Remoting code

Reply:
Yup, that makes sense. But, what is meant by "mark as restricted".

Reply:
`@Restricted(NoExternalUse.class)`